workflows/kneaddata_paired.smk

```diff
@@ -36,6 +36,22 @@ rule compressdata2:
     run: 
         shell("gzip -c {input} > {output}")
 
+checkpoint furthercompress:
+    input: "{kneadfolder}/{sample}*.fastq"
```

Comment:
I'm not sure that snakemake understands this without using a special `glob` function... does it work?